docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -116,7 +116,24 @@ Include it in your module file like this:
 .. code-block:: python
 
     #!/usr/bin/python
-    # Copyright header....
+    # Copyright © 2017 [REPLACE THIS]
```

Comment:
Invalid python :-)  I'd use (C) here.  Alternative is to use:
``` python
# -*- coding: utf-8 -*-
```
as the second line in the file.